src/pages/SignUp/index.scss

```diff
@@ -0,0 +1,46 @@
+#welcome-section {
+  .sign-up-page {
+    .email-signup-step {
+      h3 {
+        margin: 54px 6px 30px;
+        color: var(--ion-color-dark);
+        text-align: left;
+        font-weight: normal;
+      }
+      .email-box {
+        border: none;
+        border-bottom: 3px black solid;
+        margin: 0 6px 16px 6px;
+        --padding-start: 0;
+        ion-input {
+          color: var(--ion-color-medium);
```

Comment:
Probably need to do another find and replace, and probably want to get into the habit of doing it whenever you bring anything in